cmd/shell.go

```diff
@@ -435,6 +437,21 @@ func ReadConfigFile(filename string, out interface{}) error {
 	return decodeJSONStrict(file, out)
 }
 
+// ValidatorForConfigDuration enables registration of our custom config.Duration
+// type as a time.Duration and performing validation on the configured value
+// using the standard suite of validation functions.
+func ValidatorForConfigDuration(field reflect.Value) interface{} {
+	if c, ok := field.Interface().(config.Duration); ok {
+		t, err := c.ToTimeDuration()
+		if err != nil {
+			return err
```

Comment:
Does returning an error here actually work the way we want? It looks to me like [extractTypeInternal](https://github.com/letsencrypt/validator/blob/main/util.go#L13) will take that returned error and make the rest of the validator think that this struct field is supposed to be populated with an error type.

Reply:
I don't think it works. A testcase for a config.Duration with value `"60"` instead of `"60s"` fails returning from `//cmd/shell/decodeJSONStrict` which is before `validate.Struct` is called. I'm going to remove the `err` variable there.

Reply:
> A testcase for a config.Duration with value "60" instead of "60s" fails returning from //cmd/shell/decodeJSONStrict which is before validate.Struct is called.

That seems like the correct behavior to me. Someone putting "60" instead of "60s" means that they haven't provided the correct type. We'd expect it to fail in the same way -- i.e. _before_ validation -- if they put "asdf" for an integer field.